src/main/java/com/autocommunity/backend/repository/EventRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.autocommunity.backend.repository;
+
+import com.autocommunity.backend.entity.map.EventEntity;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface EventRepository extends CrudRepository<EventEntity, UUID> {
+    List<EventEntity> findAll();
```

Comment:
this line is not needed, it is declared in CrudRepository

similar in MarkerRepository